src/swap/changenow.js

```diff
@@ -314,6 +327,13 @@ export function makeChangeNowPlugin (
       const min = await get(
         'min-amount/' + quoteParams.from + '_' + quoteParams.to
       )
+      if (!min.minAmount) {
```

Comment:
This will trigger if `minAmount` is 0, which is still valid. I think it would be safer to have `min.minAmount != null`, so we only throw if the amount is really missing.